tests/Resources/ti.ui.tableview.test.js

```diff
@@ -1742,13 +1691,9 @@ describe('Titanium.UI.TableView', function () {
 		finish();
 	});
 
-	it('TableView headerView & footerView', finish => {
-
-		// FIXME: Does not honour scale correctly on macOS.
-		if (isCI && utilities.isMacOS()) {
-			return finish();
-		}
-
+	// FIXME: For an unknown reason, this test causes an 'signal error code: 11' exception on iOS
+	// shortly after running successfully.
+	it.iosBroken('TableView headerView & footerView', finish => {
```

Comment:
These tests cause a really interesting error on iOS, after the test is ran and passes the app will crash shortly after (not instantly, allows a few more tests to run). Possibly an issue with TableView headerView and footerView memory management?